src/writeData/subscriptions/components/SubscriptionCard.tsx

```diff
@@ -63,18 +121,41 @@ const SubscriptionCard: FC<Props> = ({subscription}) => {
         name={subscription.name}
         onClick={() => {
           event('subscription card clicked', {}, {feature: 'subscriptions'})
-          history.push(
-            `/orgs/${org.id}/${LOAD_DATA}/${SUBSCRIPTIONS}/${subscription.id}`
-          )
+          goToSubscriptionDetails()
         }}
         testID="subscription-name"
       />
       <ResourceCard.Description
         description={`${subscription.brokerHost}:${subscription.brokerPort}/${subscription.topic}`}
       />
       <ResourceCard.Meta>
+        {!!bulletins.length ? (
+          <Label
+            id="tid"
+            key="tkey"
+            name={`${bulletins.length} Message${
+              bulletins.length === 1 ? '' : 's'
+            }`}
+            color={InfluxColors.Grey25}
+            description={`${bulletins.length} Message${
+              bulletins.length === 1 ? '' : 's'
+            }`}
+            onClick={goToSubscriptionDetails}
+            testID="subscription-messages--label"
+          />
+        ) : (
+          <Label
+            id="tid"
+            key="tkey"
+            name="No Messages"
```

Comment:
I'm concerned that `Messages` might be confusing for the user. They might think this means that no messages have been received by the subscription from their MQTT broker. I'm thinking that `Issues` or `Notifications` might be a better work to use here

Reply:
I was just about to comment saying the same!

Reply:
i wasn't comfortable with the term, "Messages", as well. That's why I wrote as "errors/messages" in the description.